src/components/divider/index.stories.mdx

```diff
@@ -0,0 +1,31 @@
+import { Meta, Story, Canvas, ArgsTable } from "@storybook/addon-docs";
+
+import Divider from ".";
+
+<Meta title="Components/Divider" component={Divider} />
+
+# Divider
+
```

Comment:
in the ac of the ticket, there's this requirement: 

Explain this purpose in the doc’s storybook intro and prop table [aria-hidden - Accessibility | MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-hidden) [4.4 Grouping content — HTML5](https://www.w3.org/TR/2014/REC-html5-20141028/grouping-content.html#the-hr-element)

Can you add this information as well to the description? Thanks!

Reply:
Thanks @JackHowa , Fixed it in [18d538a](https://github.com/WPMedia/arc-themes-components/pull/145/commits/18d538ab116bb41d7b3d8de8b916c4ac0d029888)

Reply:
thanks for including a description. in the future, if you directly quote a text, it's a good idea to link to it and use quotation marks. I noticed that this was from w3 schools:

> The <hr> tag defines a thematic break in an HTML page (e.g. a shift of topic).
> 
> The <hr> element is most often displayed as a horizontal rule that is used to separate content (or define a change) in an HTML page.

https://www.w3schools.com/tags/tag_hr.asp

Reply:
@JackHowa added reference [e954355](https://github.com/WPMedia/arc-themes-components/pull/145/commits/e9543550fd71be386419508d41167ef6bef9ec5f)

Reply:
Thanks for paraphrasing the quoted text. Can you describe examples for when the hr tag's aria hidden should be used or when it shouldn't be used? For reference, we talked about this in a previous meeting on the hr tag. Here's the research and notes from that discussion https://arcpublishing.atlassian.net/wiki/spaces/TI/pages/3169419461/Investigation+for+tmedia+768+on+dividers+I+see+dividing+lines.+What+should+I+do#Research

Reply:
Thanks @JackHowa , Fixed it in [242d61f](https://github.com/WPMedia/arc-themes-components/pull/145/commits/242d61f53a4dcf5792b7e397bc6db9fc268ab9bc)

Reply:
Sorry, @malavikakoppula if I wasn't communicating clearly. In the ac, I was discussing when to use the aria-hidden and when not to use aria-hidden in an hr tag -- not when to use the component itself. I've included a suggestion to help out :)

Reply:
TYVM @JackHowa

Reply:
that was quick! does that make sense. can you include an example in storybook where you're showing when to use it and when not to with content?

Reply:
did you see this comment? I wanted to make sure that my documentation made sense in practice. I brought this comment up a couple times in standup. I was curious if we could put examples of content in the storybook divider stories. I would expect text or other content showing when to separate with assistive hidden and not. 

![Screen Shot 2022-10-17 at 10 56 11](https://user-images.githubusercontent.com/5950956/196225612-b1624b4a-3093-4ab5-aa6a-df85b52371a0.png)


cc @malavikakoppula @matthewroach

Reply:
Updated the story based on what I believe you are looking for?

Reply:
yep